src/components/Button/Button.scss

```diff
@@ -1,12 +1,13 @@
 .button {
-  color: white;
-  border: 1px solid transparent;
-  border-bottom-color: white;
+  color: rgba(255, 255, 255, 0.2);
+  display: block;
+  //border: 1px solid transparent;
+  //border-bottom-color: white;
```

Comment:
just remove

Reply:
It is based on how it was styled in figma